scripts/release/test_package.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import (
+    Path,
+)
+import subprocess
+from tempfile import (
+    TemporaryDirectory,
+)
+import venv
+
+
+def create_venv(parent_path: Path) -> Path:
+    venv_path = parent_path / "package-smoke-test"
+    venv.create(venv_path, with_pip=True)
+    subprocess.run(
+        [venv_path / "bin" / "pip", "install", "-U", "pip", "setuptools"], check=True
+    )
+    return venv_path
+
+
+def find_wheel(project_path: Path) -> Path:
+    wheels = list(project_path.glob("dist/*.whl"))
+
+    if len(wheels) != 1:
+        raise Exception(
+            f"Expected one wheel. Instead found: {wheels} "
+            f"in project {project_path.absolute()}"
+        )
+
+    return wheels[0]
+
+
+def install_wheel(venv_path: Path, wheel_path: Path) -> None:
+    subprocess.run(
+        [venv_path / "bin" / "pip", "install", f"{wheel_path}"],
+        check=True,
+    )
+
+
+def test_install_local_wheel() -> None:
+    with TemporaryDirectory() as tmpdir:
+        venv_path = create_venv(Path(tmpdir))
+        wheel_path = find_wheel(Path("."))
+        install_wheel(venv_path, wheel_path)
+        print("Installed", wheel_path.absolute(), "to", venv_path)
+
+        package_tests = Path("./scripts/release/tests_to_run_on_package.py")
```

Comment:
Yup, I was specifically removing the manual testing in favor of automating it. No problem to make it manual again. In theory those tests could just be added to the `tests_to_run_on_package`, but just having a terminal open to the venv would allow you mess around with it more. Will put it back to work like `py-evm`.